spec/requests/applications_spec.rb

```diff
@@ -24,13 +24,35 @@
     it 'allows admin to approve' do
       app = create(:application)
       admin_user = create(:user, admin: true)
+      deliveries.clear
       login_as(admin_user)
 
       put users_application_path(app), application: { approved: 'true' }
 
       expect(response.status).to eq(302) # redirect on success
       app.reload
       expect(app.approved?).to eq(true)
+      expect(deliveries.count).to eq(2)
+      expect(deliveries[0].subject).to eq(
+        I18n.t('dashboard.mailer.approved_application.subject', id: app.issuer)
+      )
+    end
+  end
+
+  describe 'notifications' do
+    it 'sends email to admin requesting approval' do
+      user = create(:user)
+      deliveries.clear
+      login_as(user)
+
+      post users_applications_path, application: { name: 'test' }
+
+      app = Application.last
+      expect(response.status).to eq(302)
+      expect(deliveries.count).to eq(2)
+      expect(deliveries[0].subject).to eq(
+        I18n.t('dashboard.mailer.new_application.subject', id: app.issuer)
+      )
```

Comment:
Should we also add an expectation that verifies that the email is sent to the admin (as opposed to the user, for example)?